model/src/lib.rs

```diff
@@ -47,6 +47,7 @@
 )]
 #![allow(clippy::module_name_repetitions, clippy::must_use_candidate)]
 
+pub mod applications;
```

Comment:
Module names should be singular.

```suggestion
pub mod application;
```